scripts/issue/354/upgrade_remaining_setts_pause.py

```diff
@@ -63,61 +67,109 @@
     "bcrvBadger",
 ]
 
+ACTIVE_V4_KEYS = [
+    "bveCVX",
+    "bcrvIbBTC",
+    "bcrvBadger",
+    "bbveCVX-CVX-f",
+    "bcvxCRV",
+    "bcrvTricrypto2",
+]
+
 SPECIAL_SETTS = [
     "bDIGG",
     "byvWBTC",
     "remBADGER",
     "remDIGG",
 ]
 
+UPGRADE_SIG = "upgrade(address,address)"
+SET_STRATEGIST_SIG = "setStrategist(address)"
+
+
 def main(queue="true", simulation="false"):
     if queue == "true":
         for key, address in vaults.items():
             address = web3.toChecksumAddress(address)
             if key in SETTV1_KEYS:
                 if get_implementation(address) != settV1h_logic:
-                    console.print(f"[green]Queueing upgrade on timelock for {key} to SettV1h.sol[/green]")
+                    console.print(
+                        f"[green]Queueing upgrade on timelock for {key} to SettV1h.sol[/green]"
+                    )
                     safe.badger.queue_timelock(
                         target_addr=DEV_PROXY,
-                        signature="upgrade(address,address)",
+                        signature=UPGRADE_SIG,
                         data=encode_abi(
                             ["address", "address"],
                             [address, settV1h_logic],
                         ),
                         dump_dir="data/badger/timelock/upgrade_remaining_setts/",
```

Comment:
@sajanrajdev do i remember you saying something about separate dump dirs or not..?

Reply:
I meant just separate from the other Timelock queued txs so that execute only grabs the txs for this ticket. But they can all go together.